content/en/hosting/4.x/kubernetes/_index.md

```diff
@@ -0,0 +1,16 @@
+---
+title: "Kubernetes Production Hosting in CHT 4.x"
+linkTitle: "Production Kubernetes"
+weight: 20
+---
+
+{{< hextra/hero-subtitle >}}
+  Production hosting the CHT with Kubernetes
+{{< /hextra/hero-subtitle >}}
+
+{{< callout type="warning" >}}
+Only use Kubernetes under [very specific](/hosting/kubernetes-vs-docker/) conditions. [Docker](/hosting/4.x/docker/) is recommended for most deployments.
```

Comment:
Considering that we encourage large-scale, national deployments and the biggest and most impactful by numbers CHT deployments use Kubernetes, how accurate is this sentence?

Reply:
I've removed `only` from this sentence in a9ebca9c 

We are changing what we recommend.  Now, we encourage large-scale national deployments (ie 10k+ CHWs) to run on Docker - this sentence is accurate! 

For instances that have dozens of deployments where they need to provision a multi-tenant deployment, we encourage k8s for ease of management.